plugin/trino-queryeventlistener/src/main/java/io/trino/plugin/queryeventlistener/QueryEventListenerFactory.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.queryeventlistener;
+
+import io.trino.spi.eventlistener.EventListener;
+import io.trino.spi.eventlistener.EventListenerFactory;
+
+import java.util.Map;
+
+public class QueryEventListenerFactory
+        implements EventListenerFactory
+{
+    @Override
+    public String getName()
+    {
+        return "trino-queryeventlistener";
```

Comment:
I believe this name only needs to be unique for this specific type of Plugin (EventListenerFactory) - so we should probably shorten this. The `trino-` prefix can definitely be removed and I think we should remove the `eventlistener` suffix as well. I'm looking at other plugins and this seems to be the way to go. The IcebergConnectorFactory returns `iceberg`, The FileResourceGroupConfigurationManagerFactory returns `file`, etc. So we should probably just return `query`?